VTEX - Logistics API.json

```diff
@@ -5323,392 +5328,441 @@
                                             "freightTableName": "Correios PAC",
                                             "freightTableId": "11cc4b6",
-                                            "listPrice": 10.5,
-                                            "promotionalPrice": 10.5,
-                                            "transitTime": "2.00:00:00",
+                                            "listPrice": 10.35,
+                                            "promotionalPrice": 10.35,
+                                            "timeToPresale": "00:00:00",
+                                            "transitTime": "1.00:00:00",
                                             "dockTime": "00:00:00",
-                                            "timeToDockPlusDockTime": "1.00:00:00",
-                                            "totalTime": "3.00:00:00",
-                                            "deliveryWindows": null,
-                                            "wareHouseId": "1937054",
-                                            "dockId": "1_1_1",
-                                            "wmsEndPoint": "",
+                                            "timeToDockPlusDockTime": "00:00:00",
+                                            "totalTime": "1.00:00:00",
+                                            "deliveryWindows": {
+                                                "startDateUtc": "2024-02-25T09:00:00+00:00",
+                                                "endDateUtc": "2024-02-25T12:00:00+00:00",
+                                                "listPrice": 0.0
+                                            },
+                                            "wareHouseId": "wh_temp",
+                                            "kitReservation": null,
+                                            "dockId": "dk_temp",
+                                            "wmsEndPoint": null,
                                             "location": {
-                                                "zipCode": "22220070",
+                                                "zipCode": "58039210",
                                                 "country": "BRA",
-                                                "deliveryPointId": null,
-                                                "point": null,
+                                                "point": [
+                                                    -34.829391479492188,
+                                                    -7.1152286529541016
+                                                ],
                                                 "inStore": {
-                                                    "IsCheckedIn": false,
-                                                    "StoreId": "180082"
+                                                    "isCheckedIn": false,
+                                                    "storeId": "180082"
```

Comment:
The camel case starting with the lower letter is the correct one, it is the 200 OK I got from Postman. The documentation before was incorrect or it has been changed since then.